### PR TITLE
[sortinghat] Add logging support

### PR DESCRIPTION
**Pull request by valeriocos**
This PR targets issue #149 . Thus it includes support to for logging.

### CODE REVIEW

File: sortinghat/command.py [hunk 1]
```diff
@@ -57,20 +60,22 @@ def display(self, template, **kwargs):
...
         s = t.render(**kwargs)
-
+        logger.info(s)
+        # this is left to not break the tests
         sys.stdout.write(s)
 
     def error(self, msg):
         s = "Error: %s\n" % msg
-        sys.stderr.write(s)
+        logger.error(s)
```
Comment (sduenas) [inline sortinghat/command.py]: I wouldn't change this. We have to make a difference between those messages related to the internal work of the tool (log messages) and those messages generated as an output for the user (which is this case).

Comment (valeriocos) [reply sortinghat/command.py]: ok @sduenas, I'm going to revert this change and update the PR

**Comment by valeriocos** [comment]
I have updated the pr according to your comments @sduenas , please have a look

File: sortinghat/command.py [hunk 2]
```diff
@@ -57,7 +60,8 @@ def display(self, template, **kwargs):
 
         t = env.get_template(template)
         s = t.render(**kwargs)
-
+        logger.info(s)
```
Comment (sduenas) [inline sortinghat/command.py]: This should be also removed as you did with the errors.

**Comment by valeriocos** [comment]
I have updated the PR, including changes due to the inclusion of #155 to the codebase.

**Pull request closed without merging**